addons/ingress-nginx/addon.rb

```diff
@@ -22,24 +24,10 @@
     apply_resources(
       configmap: config.configmap || {},
       node_selector: config.node_selector || {},
-      image: image_name,
```

Comment:
This essentially makes the default backend image non-configurable. Why not just pass the `config.default_backend[:image]`, that should default correctly to `registry.pharos.sh/...`

Reply:
Good catch, fixed.